scripts/mkrepo.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+# Take the deb and bundle it into a apt repository
+if [[ $# -lt 1 ]]
+then
+    echo "Usage $0 <config dir>"
+    exit
+fi
+
+confdir=$1
+
+set -x
```

Comment:
`set -ex` also works as one line, if you want to embrace the Smallest Script Wins part of #bash